1228-landing_page/owner_navigates_to_the_landing_page_for_their_registration.feature

```diff
@@ -42,7 +42,7 @@ Feature: Owner navigates to the Landing Page for their Resource
         And an Approval Request is sent to the Curator
```

Comment:
Kan være greit å være eksplisitt på hva som er "locked" over her. Jeg tolker `Resource's content is locked` som at alt skal være låst

Reply:
HAr lagt til detaljer